### PR TITLE
Kit calculator

### PR DESCRIPTION
**Pull request by oliaboo**

### CODE REVIEW

File: lesson 6/Guess.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+            print('\nCongratulations! You win!')
+            print('\nNumber of tries: ', self.counter)
+            game = Guess()
+            self.counter = 0
+        else:
+            print('\nTry again!')
+
+game = Guess()
+
+game.counter = 0
```
Comment (oliaboo) [inline lesson 6/Guess.py]: should be initialized in __init__, I guess

File: lesson 6/Guess.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+while True:
+
+    input_string = int(input('\nType your number: ')) 
+
+    if input_string == 'exit':
+
+        break
+
+        game.counter = 0
```
Comment (oliaboo) [inline lesson 6/Guess.py]: update counter only via setter, get via getter methods

File: lesson 6/Guess.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        else:
+            print('\nTry again!')
+
+game = Guess()
+
+game.counter = 0
+
+while True:
+
+    input_string = int(input('\nType your number: '))
```
Comment (oliaboo) [inline lesson 6/Guess.py]: input_string should be the attribute of class object as well

File: lesson 6/Guess.py [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+    input_string = int(input('\nType your number: ')) 
+
+    if input_string == 'exit':
+
+        break
+
+        game.counter = 0
+
+        del game
```
Comment (oliaboo) [inline lesson 6/Guess.py]: no need for this, destructor will be called anyway befor the program ends execution

File: lesson 6/Guess.py [hunk 5]
```diff
@@ -0,0 +1,41 @@
+import random
+
+class Guess:
```
Comment (oliaboo) [inline lesson 6/Guess.py]: better GuessingGame etc.

File: Class Calculator.py [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+    # This method divides two numbers
+    def divide(self):
+            try:
+                    self.x / self.y
+            except ZeroDivisionError:
+                    return 'Division by zero'
+            return self.x / self.y
+        
+
+def calculation(raw_expression):
```
Comment (oliaboo) [inline Class Calculator.py]: logic should be performed in the class

Comment (OlehKitGit) [reply Class Calculator.py]: I did it.

**Comment by OlehKitGit** [comment]
Done.

**Pull request closed without merging**